components/dev-site-notification.tsx

```diff
@@ -0,0 +1,32 @@
+import { Anchor, Dialog, Text } from "@mantine/core";
+import { useDisclosure } from "@mantine/hooks";
+import { useEffect } from "react";
+
+const DevSiteNotification = () => {
+  const [opened, { open, close }] = useDisclosure(false);
+  useEffect(() => {
+    if (window.location.hostname !== "coh3stats.com") {
```

Comment:
We can use this function https://github.com/cohstats/coh3-stats/blob/master/config.ts#L31

Reply:
Also we could use that function in in the app directly in the rendering:
```
{isDevEnv() && <DevSiteNotification />} 

```

So it's not even rendered on the main page! :)

Reply:
done